### DISCUSSION TITLE
Bold text

### DISCUSSION THREAD

**Discussion by itsNiccoloSabatini** [emilk/egui#4236] [Ideas]
please add a way that using egui::RichText its possible to make the text **Bold**.

**Comment by YgorSouza** [comment]
Would probably require #3218. Right now you can add a bold font by following [this example](https://github.com/emilk/egui/tree/4d4cb3d20db0517e74ca6ab6d3b646f78fd8123b/examples/custom_font), and then tell the `RichText` to use it via the [font](https://docs.rs/egui/0.27.0/egui/widget_text/struct.RichText.html#method.font) method.
